editor.planx.uk/src/components/MoreInfoFeedback.tsx

```diff
@@ -27,41 +27,114 @@ const FeedbackBody = styled(Box)(({ theme }) => ({
 }));
 
 const MoreInfoFeedbackComponent: React.FC = () => {
-  return (
-    <>
+  enum FeedbackView {
+    YesNo,
+    Input,
+    ThankYou,
+  }
+
+  enum FeedbackSentimentOption {
+    Yes,
+    No,
+  }
+
+  const [currentFeedbackView, setCurrentFeedbackView] = useState(
+    FeedbackView.YesNo,
+  );
+  const [feedbackOption, setFeedbackOption] =
+    useState<FeedbackSentimentOption | null>(null);
+
+  const handleFeedbackOptionClick = (event: FeedbackSentimentOption) => {
+    switch (event) {
+      case FeedbackSentimentOption.Yes:
+        setCurrentFeedbackView(FeedbackView.Input);
+        setFeedbackOption(FeedbackSentimentOption.Yes);
+        break;
+
+      case FeedbackSentimentOption.No:
+        setCurrentFeedbackView(FeedbackView.Input);
+        setFeedbackOption(FeedbackSentimentOption.No);
+        break;
+    }
+  };
+
+  const handleFeedbackFormSubmit = (e: any) => {
```

Comment:
I know this is a UI PR but just as a heads up for when you come to wire this up in a future PR - 
 - We generally use [Formik](https://formik.org/) to build and handle forms
 - We use [Yup](https://github.com/jquense/yup) when we need a schema to validation (I'd love to switch to [Zod](https://zod.dev/) but Yup and Formik play together nicely out of the box and this project pre-dates Zod)
 
We can do a handover / run though before you come to this if you'd like - there are plenty of good examples throughout the codebase also.